skins/laika/src/scss/overrides.scss

```diff
@@ -68,6 +68,25 @@
 	}
 }
 
+.campaigns--anon-form-display--two-steps {
+
+	.radio-container {
+		display: flex;
+	}
+
+	.b-radio.radio {
+		width: 180px;
+		margin-right: 18px;
+	}
+}
+
+.campaigns--anon-form-display--address-type {
+	/* Remove margin added by buefy, since each button is on its own line */
+	.b-radio.radio + .radio {
+		margin-left: 0;
+	}
+}
+
```

Comment:
I would suggest to group this with nesting so it's easier to see what belongs to this test:

```
.campaigns {
    &--anon-form-display {
         &--two-steps {
           // ...
         }
         &--address-type {
           // ...
         }
    }
}
```